draft-edm-protocol-greasing.md

```diff
@@ -198,6 +198,17 @@ not limited to those discussed in this document, can affect fingerprinting. A
 deeper analysis of this topic has been deemed out of scope.
 
 
+While protocol features, extensions, and versions all have legitimate uses, they
+can become a burden when used to excess. For example, the ability to send
+protocol grease that a peer is required to ignore can be abused to cause it to
```

Comment:
Maybe explain some examples of the abuse — not just the fact of ignoring one greased object, but having a flood of them that are all ignored, etc.

Reply:
Thought: There is a "cost" when the greasing creates state, even if only to ignore the greased value. (I can think of cases where seeing a new value needs the receiver to do work to ignore. )